Task_DEV_5/Task_DEV_5/EntryPoint.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.IO;
+
+namespace task_DEV_5
+{
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                string fileName = args[0];
+                int numberOfLines = int.Parse(args[1]);
+
+                FileGenerator fileGenerator = new FileGenerator(Path.GetFullPath(@"..\\..\\" + fileName), numberOfLines);
```

Comment:
You could do it in some separate variable `@"..\\..\\" + fileName`.

Reply:
Because if you decide to change path you could change only variable with the path but not all places where this path is used.

Reply:
Done.